packages/react-pagination/docs/react-pagination.stories.js

```diff
@@ -0,0 +1,30 @@
+import React from "react";
+import { Pagination } from "../lib/react-pagination";
+
+export default {
+	title: "Components/Pagination",
+	component: Pagination,
+};
+
+let childrenArray = new Array();
+for (let i = 1; i <= 34 /* from here you can change the number of elements that can be inputed */; ++i)
+	childrenArray.push(<h5 key={i}>This is the sample data that I am working with. Id number is {i}</h5>);
+
+export const primary = args => (
+	<Pagination {...args}>
+		<div>Page</div>
+	</Pagination>
+);
+
+primary.storyName = "Pagination";
+primary.args = {
+	limit: 5, //elements per page
+	results: true, //number of results
+	skip: true, //skip to last and first page buttons status
```

Comment:
@Gowtham369 found a bug related to skip buttons. When I set them to `false` the 3 dots button appearance is inconsistent.

**When moving "forward"**
1. Page #1 disappears as soon as I reach page #5.
2. The "three dots" button indicating there are more hidden pages is nowhere to be found.

**When moving "backward"**
1. Page #7 disappears as soon as I reach page #3.
2. The "three dots" button indicating there are more hidden pages is nowhere to be found.

Just review your logic again to show/hide the "three dots" button where there are more pages that the user doesn't see, but be careful on breaking the logic for the skip buttons.

<img width="407" alt="Screen Shot 2021-06-09 at 19 18 37" src="https://user-images.githubusercontent.com/2328848/121445647-807dd500-c957-11eb-8839-d31f85efdd40.png">

Reply:
Actually it is not the case only with `skip:false;` .
It flows the same for the `skip:true;` as well.
I put the logic thinking that if there is only one page hidden on either ends, then it won't look good.
Anyways I will tweak it, such that it will show elipsis incase 1 page also.